src/main/scala/rna16s.scala

```diff
@@ -46,7 +46,10 @@ case object rna16s extends AnyBlastDB {
   /* Here we want to keep sequences which */
   val predicate: (Row, FASTA.Value) => Boolean = { (row, fasta) =>
      /* - are annotated as encoding 16S */
-     (row.select(gene_name) == geneNameFor16S) &&
+     (row.select(gene_name) == geneNameFor16S ||
+      fasta.getV(header).description.toLowerCase.contains("16s") ||
+      fasta.getV(header).description.toLowerCase.contains("small subunit ribosomal rna")
```

Comment:
1. we take everything from RNACentral
2. filter the table, leaving only those rows that refer to the sequences in the FASTA file (i.e. only those that we can use for Blast)
3. then for this particular 16S database we apply this filter:
  - choosing only 16s:
    + those sequences that have 16S rRNA annotation in the _RNACentral table_
    + **OR** those that have 16s or 16S in their FASTA _header description_ **OR**
    + **OR** and those that contain small subunit ribosomal in their _header description_
  - **AND** the sequence has length > 1000
  - **AND** are not in [this](https://github.com/era7bio/db.rna16S/blob/master/src/main/scala/rna16s.scala#L24-L42) list